src/main/java/co/elastic/logstash/filters/elasticintegration/geoip/IpDatabaseAdapter.java

```diff
@@ -8,8 +8,12 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.elasticsearch.logstashbridge.core.CheckedBiFunctionBridge;
 import org.elasticsearch.logstashbridge.geoip.IpDatabaseBridge;
-import org.elasticsearch.logstashbridge.geoip.MaxMindDbBridge;
+import org.elasticsearch.ingest.geoip.shaded.com.maxmind.db.CHMCache;
+import org.elasticsearch.ingest.geoip.shaded.com.maxmind.db.NoCache;
+import org.elasticsearch.ingest.geoip.shaded.com.maxmind.db.NodeCache;
+import org.elasticsearch.ingest.geoip.shaded.com.maxmind.db.Reader;
```

Comment:
Ah, this was my confusion that we fully move to the ES logstash-bridge w/o embedding the JARs but after analyzing the risk of conflict, as we took offline discussion, our original approach is the right one. 

This indicates we DON'T NEED maxmind dependencies in
- `libs/logstash-bridge/build.gradle` -> `compileOnly('com.maxmind.db:maxmind-db:3.1.1')`
- and  `libs/logstash-bridge/src/main/java/module-info.java` -> `requires com.maxmind.db;`

Reply:
I will remove these dependencies, just noting here to remind myself while reviewing.